### PR TITLE
fix: Hide finished boosted quests after 3 weeks of expiry

### LINKED ISSUE #520: Hide finished boosted quests
telegram-cloud-photo-size-4-6003631507303939840-y

These quests expired a long time ago, we need to hide them in some way. I propose after 3 weeks of claiming period. 

We hide the quest from this page
Every boost on this page has an expiry which you can check from the API response and you can use that to check how much time has passed since the expiry

expected time: Should be around 2 hours

### PR DESCRIPTION
**Pull request by Oshioke-Salaki**
closes #520 

#Pull Request Type
Hide finished boosted quests

Please add the labels corresponding to the type of changes your PR introduces:

1. Bugfix

Resolves: #520 By hiding quest boosts that have exceeded 3 weeks of expiry.

### CODE REVIEW

File: package-lock.json
Comment (ayushtom) [inline package-lock.json:1]: remove this change if you aren't making any changes to package json file

Comment (Oshioke-Salaki) [reply package-lock.json:1]: i have fixed the package-lock.json issue

Comment (ayushtom) [inline package-lock.json:1]: i still see this file. Can you remove it from the changes so that only the quest boost page changes are included in the PR

Comment (Oshioke-Salaki) [reply package-lock.json:1]: I think i will just open a new pull request.

**Pull request closed without merging**